cypress/integration/attributes/check-cluster-attributes-singleton-in-zigbee-mode.spec.js

```diff
@@ -0,0 +1,115 @@
+/// <reference types="cypress" />
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false
+})
+
+// Description:
+//   In Zigbee, endpoints that implement the same Cluster have special
+// effects on the Cluster specific attributes and commands.
+// The attributes and commands behave as global variables. There is only
+// one global cluster specific attribute or command value/setting shared across endpoints.
+// They are not endpoint specific.
+
+describe('Testing endpoints sharing attribute values', () => {
+  it('create Endpoint 1', () => {
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+    cy.gotoAttributePage('Billing Unit (0x0203)', 'General')
+  })
+  it(
+    'set initial values on Endpoint 1',
+    { retries: { runMode: 2, openMode: 2 } },
+    () => {
+      // Set ZCL version - 1
+      cy.get(
+        ':nth-child(1) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      )
+        .clear({ force: true })
+        .type('1', { force: true })
+
+      // Set power source - 22
+      cy.get(
+        ':nth-child(8) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      )
+        .clear({ force: true })
+        .type('22', { force: true })
+
+      // Set cluster revision - 33
+      cy.get(
+        ':nth-child(19) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      )
+        .clear({ force: true })
+        .type('33', { force: true })
+
+      // Set ResetToFactoryDefaults command
+      cy.gotoCommandsTab()
+      cy.get(':nth-child(3) > .q-mt-xs > .q-checkbox__inner > input').check({
+        force: true,
+      })
+    }
+  )
+  it('create Endpoint 2', () => {
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+
+    cy.gotoAttributePage('Chatting Station (0x0601)', 'General')
+  })
+  it(
+    'verify initial values on Endpoint 2 are reflected from Endpoint 1',
+    { retries: { runMode: 2, openMode: 2 } },
+    () => {
+      // Set ZCL version - 1
```

Comment:
Get from Ep1 instead of set?

Reply:
The chained .type() API call actually does the set.